radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/core/api.yaml

```diff
@@ -1017,11 +1044,6 @@ tags:
       the validators performing consensus. As a consequence of this, it is _not_ guaranteed to be increasing, and it
```

Comment:
I think you explained that the timestamp is actually the median - we should probably mention this. (It's certainly safer than the mean)

Reply:
Would rather not specify how it's derived at this time as I would like to see the implementation of this change actually in the near future.